.gitignore

```diff
@@ -16,3 +16,4 @@ rendered
 particles_*
 sim_*
 *.bin
+.vscode/
```

Comment:
Leave an EOL here?

Reply:
done, but not sure why git-hub is 'hiding' the EOL
![image](https://user-images.githubusercontent.com/48085697/135461776-6eda27bf-213e-488a-8515-a3b7473942ee.png)